MoodCookie/app/src/main/java/cecelia/moodcookie/ConfirmationPageFragment.java

```diff
@@ -14,7 +14,7 @@
 
 public class ConfirmationPageFragment extends Fragment {
 
-    private CameraInterface mInterface;
+    private MainActivity mainActivity;
```

Comment:
This was originally an interface that the Main Activity implemented to give fragments a way to invoke its methods. I realized that since we only had one Activity it was easier to just cast the return from getActivity() to a Main Activity type